packages/app-mobile/components/FolderPicker.tsx

```diff
@@ -0,0 +1,91 @@
+const React = require('react');
+
+import { FunctionComponent } from 'react';
+import { _ } from '@joplin/lib/locale';
+import Setting from '@joplin/lib/models/Setting';
+import Folder, { FolderEntityWithChildren } from '@joplin/lib/models/Folder';
+const { themeStyle } = require('./global-style.js');
+import Dropdown, { DropdownListItem, OnValueChangedListener } from './Dropdown';
+import { FolderEntity } from '@joplin/lib/services/database/types';
+
+interface FolderPickerProps {
+	disabled?: boolean;
+	selectedFolderId?: string;
+	onValueChange?: OnValueChangedListener;
+	mustSelect?: boolean;
+	folders: FolderEntity[];
+	placeholder?: string;
+	darkText?: boolean;
+	themeId?: string;
+}
+
+
+const FolderPicker: FunctionComponent<FolderPickerProps> = ({
+	disabled,
+	selectedFolderId,
+	onValueChange,
+	mustSelect,
+	folders,
+	placeholder,
+	darkText,
+	themeId = Setting.value('theme'),
```

Comment:
Please avoid making `themeId` optional (see [the style guide](https://github.com/laurent22/joplin/blob/dev/readme/coding_style.md#avoid-default-and-optional-parameters)).

Reply:
Ok fixed this